package.json

```diff
@@ -4,7 +4,7 @@
   "description": "EPAM-JSA-AGATE --Local Business Finder (Yelp)",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "mocha $(find test/mocha -name '*js')"
```

Comment:
task name should be `test:backend` also just tell mocha which folder is where your backend tests are